packages/aetna/assets/styles/components/_media.scss

```diff
@@ -1,13 +1,13 @@
-.alignleft {
+#content .alignleft {
```

Comment:
Is there a need to keep these classes outside of #content as well?

Reply:
Found an example of using alignLeft in dashboard's id=wpcontent
![image](https://user-images.githubusercontent.com/21694293/90694960-87601d80-e247-11ea-893d-9d44a6cc7b21.png)